index.html

```diff
@@ -12,7 +12,13 @@
     <script src="src/index.js" defer></script>
   </head>
   <body>
-    <div class="wrapper">
+    <div id="loader">
+      <iframe
```

Comment:
to jest mega zjebane, wiem, ale ja chciałam sobie tego gifa wrzucić w komponencie Loader to tak wariował, że nie szło go uspokoić. Jeżeli ktoś z Was ma ochotę - zapraszam, jeżeli nie - mogę po prostu wrzucić zwykły loader cssowy i nie będzie tego problemu